specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/azureFirewall.json

```diff
@@ -736,7 +736,8 @@
          "description":"The action type of a rule collection",
          "enum":[
             "Allow",
-            "Deny"
+            "Deny", 
+            "Alert"
```

Comment:
Please revert this change. "Alert" value has been removed by AzureFirewall team https://github.com/Azure/azure-rest-api-specs/pull/5428